WRQ-9365: Added VirtualGridList sample to load data incrementally for infinite scrolling (#225)
enactjs/samples: sandstone/pattern-virtualgridlist-infinite-scroll/README.md

## VirtualGridList With Infinite Scroll Pattern

A sample Enact application that shows off how to load items with infinite scroll in VirtualGridList.

Run `npm install` then `npm run serve` to have the app running on [http://localhost:8080](http://localhost:8080), where you can view it in your browser.

#### Enact Components Used
- `sandstone/ImageItem`
- `sandstone/Panels/Header`
- `sandstone/Panels/Panel`
- `sandstone/VirtualList/VirtualGridList`

> In this app, load data when scrolling reaches the bottom of the virtualgridlist. Use your API server to data fetch to load the data.

Reviewer: ```suggestion
In this app, data is loaded when scrolling reaches the bottom of the virtualgridlist. Use your API server to fetch data to load.
```

> Performance may decrease when using the onScroll handler, so use the onScrollStop handler like in the sample.

Reviewer: ```suggestion
Performance may decrease when using the `onScroll` handler, so use the `onScrollStop` handler like in the sample.
```

> ## VirtualGridList With Infinite Scroll Pattern
> 
> A sample Enact application that shows off how to load items with infinite scroll in VirtualGridList.
> 
> Run `npm install` then `npm run serve` to have the app running on [http://localhost:8080](http://localhost:8080), where you can view it in your browser.
> 
> #### Enact Components Used
> - `sandstone/ImageItem`
> - `sandstone/Panels/Header`
> - `sandstone/Panels/Panel`
> - `sandstone/VirtualList/VirtualGridList`
> 
> In this app, data is loaded when scrolling reaches the bottom of the virtualgridlist. Use your API server to fetch data to load.
> Performance may decrease when using the `onScroll` handler, so use the `onScrollStop` handler like in the sample.
> 
> ---
> 
> This project was bootstrapped with the Enact [cli](https://github.com/enactjs/cli).

Reviewer 2: ```suggestion
## Incremental Data Loading in VirtualGridList Pattern

A sample Enact application that demonstrates how to load items incrementally in a VirtualGridList.

Run `npm install` then `npm run serve` to have the app running on [http://localhost:8080](http://localhost:8080), where you can view it in your browser.

#### Enact Components Used
- `sandstone/ImageItem`
- `sandstone/Panels/Header`
- `sandstone/Panels/Panel`
- `sandstone/VirtualList/VirtualGridList`

In this app, data is loaded as you scroll to the bottom of the VirtualGridList. Use your API server to fetch data to load.
Performance may decrease when using the `onScroll` handler, so use the `onScrollStop` handler as shown in the sample.

---

This project was bootstrapped with the Enact [cli](https://github.com/enactjs/cli).
```
